.github/workflows/frontend-mobile.yml

```diff
@@ -0,0 +1,29 @@
+name: Flutter lint and smoke test
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
```

Comment:
Maybe we should only run these actions if we find changes below /frontend-mobile to prevent obsolete runs. Same with backend action

Reply:
Yes. I just copied the triggers for now, but we should discuss when we really want to run these. probably only  on merge requests?